interactions/api/http/channel.py

```diff
@@ -93,7 +93,7 @@ async def create_channel(
         """
         Creates a channel within a guild.
 
-        ..note::
+        .. note::
             This does not handle payload in this method. Tread carefully.
```

Comment:
Treat*?

Reply:
excerpt from the Plain English website.

![image](https://user-images.githubusercontent.com/41456914/166316064-080326a6-1fd4-4a76-b5e9-550ea2fc88fd.png)

Reply:
Cool thanks, I translated it to german so yeah xd